playbooks/roles/edxapp/tasks/deploy.yml

```
@@ -216,6 +216,22 @@
     - install
     - install:app-requirements
 
+- name: "Pull translations using Atlas after Python dependencies installed"
+  shell: |
+    set -eu -o pipefail
+    source {{ edxapp_venv_dir }}/bin/activate
+    # Use production Django settings because otherwise debug_toolbar will be referenced and cause
+    # an error (we don't have developer Python deps installed.) Use minimal configs because the
+    # real configs aren't installed until later in the playbook.
+    DJANGO_SETTINGS_MODULE=lms.envs.production LMS_CFG=lms/envs/minimal.yml STUDIO_CFG=lms/envs/minimal.yml \
+      OPENEDX_ATLAS_PULL=true make pull_translations
```

I've made some suggestions, but please do not accept it as-is. My Ansible skills are pretty rusty at the moment.


```suggestion
  shell: |
    set -eu -o pipefail
    source {{ edxapp_venv_dir }}/bin/activate
    make pull_translations
  environment:
    # Use production Django settings because otherwise debug_toolbar will be referenced and cause
    # an error (we don't have developer Python deps installed.) Use minimal configs because the
    # real configs aren't installed until later in the playbook.
    EDX_PLATFORM_SETTINGS: "production"
    LMS_CFG: "lms/envs/minimal.yml"
    STUDIO_CFG: "lms/envs/minimal.yml"
    OPENEDX_ATLAS_PULL: "true"
```

---

Oh nice, I didn't know Ansible had `environment` as a field on tasks in general! Also, I can get rid of the `set -eu -o pipefail` since I no longer have a complicated shell script; the relevant command is the last one.

I'll give EDX_PLATFORM_SETTINGS a shot as well.